course/src/isometry.cc

```diff
@@ -0,0 +1,83 @@
+/**
+ * @file Vector.c
+ * @author your name (you@domain.com)
+ * @brief
+ * @version 0.1
+ * @date 2020-02-03
+ *
+ * @copyright Copyright (c) 2020
+ *
+ */
+
+#include <isometry.h>
+
+namespace ekumen {
+namespace math {
+Vector3::Vector3(const double &x, const double &y, const double &z) : x_(x), y_(y), z_(z) {}
+
+Vector3::Vector3() {
+    x_ = kZero.x_;
+    y_ = kZero.y_;
+    z_ = kZero.z_;
+}
+
+Vector3 Vector3::cross(Vector3 v) const {
+    auto i = (y_ * v.z_) - (v.y_ * z_);
+    auto j = (x_ * v.z_) - (v.x_ * z_);
+    auto k = (x_ * v.y_) - (v.x_ * y_);
+
+    return Vector3(i, j, k);
+}
+
+const double &Vector3::operator[](int val) const {
+    switch (val) {
+        case 0:
+            return x_;
+            break;
```

Comment:
Since you called return above, break can be avoided (linters will complain that it's unreachable code).